app/controllers/api/metric_rollups_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class MetricRollupsController < BaseController
+    REQUIRED_PARAMS = %w(resource_type capture_interval start_date).freeze
```

Comment:
@jntullo I think that would go a long way towards limiting the scope of data we will return. However, I think that we'll want to be able to be able to request all hourly metrics for all VMs, for a specific time range, for instance. This is common on the reporting side. Requiring resource_ids will make that more difficult.